samples/Azure.ResourceManager.Sample/Generated/Models/DiskEncryptionSetParameters.cs

```diff
@@ -10,7 +10,7 @@
 namespace Azure.ResourceManager.Sample
 {
     /// <summary> Describes the parameter of customer managed disk encryption set resource id that can be specified for disk. &lt;br&gt;&lt;br&gt; NOTE: The disk encryption set resource id can only be specified for managed disk. Please refer https://aka.ms/mdssewithcmkoverview for more details. </summary>
-    public partial class DiskEncryptionSetParameters : Core.SubResource
+    public partial class DiskEncryptionSetParameters : SubResourceWritable
```

Comment:
>SubResourceWritable [](http://example.com/codeflow?start=55&length=19)

nit: SubResourceWritable or WritableSubResource? I think Adj should proceed the noun.

Reply:
I will add an ADO item for that as it will require changes in azure-sdk-for-net

Reply:
https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5844